libcxx/test/std/utilities/format/format.arguments/format.arg.store/make_format_args.verify.cpp

```diff
@@ -0,0 +1,19 @@
+//===----------------------------------------------------------------------===//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+
+// <format>
+
+// template<class... Args>
+//   format-arg-store<format_context, Args...>
+//   make_format_args(const Args&... args);
+
+#include <format>
+
+// expected-error@*:* {{no matching function for call to 'make_format_args'}}
+[[maybe_unused]] auto store = std::make_format_args(42);
```

Comment:
This should be a SFINAE test instead. You could actually put this in the `.pass.cpp` test for `make_format_args` and do:

```
template <class ...Args>
concept can_make_format_args = requires (Args&& ...args) {
  std::make_format_args(std::forward<Args>(args)...);
};

static_assert(can_make_format_args<int&>); // test the test
static_assert(!can_make_format_args<int>);
static_assert(!can_make_format_args<int&&>);
```